### PR TITLE
bug_: fix resend type for private group messages

### PR DESCRIPTION
**Pull request by cammellos**
The resend type was wrong for private group chats, leading for those messages not being retried anymore after coming back online.

### CODE REVIEW

File: protocol/chat.go [hunk 1]
```diff
@@ -288,6 +288,14 @@ func (c *Chat) IsActivePersonalChat() bool {
 	return c.Active && (c.OneToOne() || c.PrivateGroupChat() || c.Public()) && c.CommunityID == ""
 }
 
+func (c *Chat) DefaultResendType() common.ResendType {
```
Comment (ilmotta) [inline protocol/chat.go:296]: @cammellos, the function `protocol#GetResendTypeForChat` seems to be a duplicate of this new function you just defined. Shouldn't the places `GetResendTypeForChat` is called also use the new implementation you defined that caters for private group chats? If so, probably better would be to remove/replace `GetResendTypeForChat` with your `DefaultResendType`?

Comment (cammellos) [reply protocol/chat.go:296]: oh yes, I didn't notice it was already there, I assumed it wasn't, I'll use mine since it's tested :)

**Review by ilmotta — APPROVED** [review]

**Review by jrainville — APPROVED** [review]
LGTM

**Review by igor-sirotin — APPROVED** [review]

**Review by qfrank — APPROVED** [review]
thank you for the fix!

**Pull request merged**